app/src/main/java/bg/zahov/app/data/model/Sound.kt

```diff
@@ -6,14 +6,17 @@ class SoundKeys {
         const val SOUND_1 = "cool_name_1"
         const val SOUND_2 = "cool_name_2"
         const val SOUND_3 = "cool_name_3"
+
+        val sounds = listOf(SOUND_1, SOUND_2, SOUND_3)
```

Comment:
Instead of declaring constants + a list of them you can use an `enum class` and use its `entries`. Same goes for the other similar usages in the PR.

Reply:
remade the whole settings system. Now everything is with enums